tests/std/tests/GH_002030_asan_annotate_string/test.cpp

```diff
@@ -1867,6 +1981,7 @@ void run_allocator_matrix() {
     run_custom_allocator_matrix<CharType, aligned_allocator>();
     run_custom_allocator_matrix<CharType, explicit_allocator>();
     run_custom_allocator_matrix<CharType, implicit_allocator>();
+    run_asan_disablement_test();

```

Comment:
TODO: I suppose I should add template-paramterize this new test as well. I'll tackle that tomorrow

Reply:
addressed, resolving.